packages/sources/cfbenchmarks-test/src/config/index.ts

```diff
@@ -44,7 +44,7 @@ export const customSettings = {
     description: 'Toggle to set the default endpoint to use WebSockets',
     type: 'boolean',
     required: false,
-    default: true,
+    default: false,
```

Comment:
Can you check the validation method below? Seems like that would prevent this?

Reply:
Done